### PR TITLE
Sn 263: Reformat code, Refactor `users` to `annotators` and `reviewers` wherever applicable

### PR DESCRIPTION
**Pull request by anuran-roy**
# Description

Changed `user` to `annotator` and `reviewer` wherever applicable. 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [X] Tested the API endpoint on Swagger with the 'Try it out' feature
- [X] Tested the Models using the Django Shell
- [X] Any other tests you performed for checking edge cases?
- [X] Any other tests you performed for checking exceptions?

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: backend/tasks/views.py [hunk 1]
```diff
@@ -93,14 +93,14 @@ def annotations(self, request, pk):
         task = self.get_object()
         annotations = Annotation.objects.filter(task=task)
         project = Project.objects.get(id=task.project_id.id)
-        user = request.user
+        annotators = request.user
```
Comment (GokulNC) [inline backend/tasks/views.py:96]: Shouldn't this just be `annotator` ?

Comment (anuran-roy) [reply backend/tasks/views.py]: Sorry for the overlook :\

Fixed!

**Review by ishvindersethi22 — APPROVED** [review]
To be tested on dev server.
